CHANGES.rst

```diff
@@ -1,10 +1,13 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-10)
 --------------------------
 
 - Fixes problem on GitLab integration preventing the synchronisation of projects with existing webhooks.
+- Fixes restarting of Yadage and CWL workflows.
+- Fixes Celery version less that 5 because of conflicting REANA-Commons' ``kombu`` dependency.
```

Comment:
We don't really fix Celery here, so perhaps reformulate as:

- Fixes conflicting `kombu` installation requirements by requiring Celery version 4.